app/tsconfig.json

```
@@ -12,14 +12,14 @@
 		"resolveJsonModule": true,
 		"sourceMap": true,
 		"baseUrl": ".",
-		"types": ["vite/client", "jest"],
+		"types": ["vite/client"],
 		"paths": {
 			"@/*": ["src/*"],
 			"preact/*": ["stub/empty.d.ts"],
 			"tslib": ["node_modules/tslib/tslib.d.ts"]
 		},
 		"lib": ["esnext", "dom", "dom.iterable", "scripthost"]
 	},
-	"include": ["src/**/*.ts", "src/**/*.vue"],
+	"include": ["src/**/*.ts", "src/**/*.vue", "tests/**/*.ts"],
```

```suggestion
	"include": ["src/**/*.ts", "src/**/*.vue"],
```

Not required anymore